ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+## Operating system
```

Comment:
Add in a header saying what this is for (bugs) and what it's not, like 'If you have a question, please don't use this form, but instead ask on the Synapse Help Forum <https://www.synapse.org/#!SynapseForum:default>'.